cassiopeia/solver/DistanceSolver.py

```diff
@@ -250,7 +228,11 @@ def update_dissimilarity_map(
         """
         pass
 
-    def append_sample_names(self, solution: nx.DiGraph) -> nx.DiGraph:
+    def append_sample_names(
```

Comment:
Is this method private? (If so let's use an underscore)

Reply:
Good point.